pkg/krok/providers/livestore/user_store.go

```diff
@@ -146,16 +148,18 @@ func (s *UserStore) getByX(ctx context.Context, log zerolog.Logger, field string
 		DisplayName: storedDisplayName,
 		ID:          storedID,
 		APIKeys:     apiKeys,
-		LastLogin:   storedLastLogin}, nil
+		LastLogin:   storedLastLogin,
+		Token:       storedToken.String,
```

Comment:
We should somehow mark this field so we don't display it ever.
I'm not afraid that someone hacks this, but it's usless information for the user. Also, it's theoretically possible to iterate through all comibnations and find one that matches this string even though it's 60^32 combinations. :)

Reply:
So I have made sure its ignored in the JSON annotation (`json:"-"`) for the model. Is that enough?

Reply:
Yeah I think so. Thanks.